src/rwe/cob/CobExecutionContext.cpp

```diff
@@ -750,7 +750,7 @@ namespace rwe
         }
     }
 
-    unsigned int CobExecutionContext::nextInstruction()
+    int CobExecutionContext::nextInstruction()
```

Comment:
instruction opcodes are also probably better left as unsigned? Since they go into a big switch to compare against the opcode literals in `OpCode` which do make use of the high bit

Reply:
yeah in retrospect I should have left most of the cob stuff alone - I'll change these

Reply:
in fact a few of the headers specify uint32_t so I'll use that explicitly for the returns where it makes sense. Easier to read, and usually when I see those I think of bit masks etc which I think is the right thing here (let me know if I'm wrong)